frontend/gateway/pb/exit.go

```diff
@@ -1,6 +1,11 @@
-package errdefs
+package moby_buildkit_v1_frontend //nolint:golint
```

Comment:
Not sure why you moved this here.  I think this can live in the original `gateway/errdefs` package, at least I vaguely recall that was the original preference.  I think this error would be similar to the several errors in `solver/errdefs` that ToProto defined.

Somewhat related discussion on where the error lives: https://github.com/moby/buildkit/pull/1830

Reply:
The proto object needs to have a `WrapError` method which returns a `*ExitError`, and `*ExitError` needs a `ToProto` method that returns the proto object. If they lived in different packages, wouldn't this imply cyclic imports?